tasks/js-dom.md

```diff
@@ -10,6 +10,18 @@ Learn some frontend basics: DOM API
 - [JavaScript and the DOM](https://classroom.udacity.com/courses/ud117)
 (4 lessons, estimated completion time 8 hrs)
 - [freecodecamp Algorithm Scripting Challenges](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting) - finish second half of the course (10 lessons) and save a screenshot.
+- **Practical task**: Implement interactive side-menu' site without page reloads aka simple SPA. Theme may be any: pokemons, phones, site pages aka about/prices/goal/etc.
```

Comment:
I think `You may chose any theme:` is better here

Reply:
done

Reply:
my mistake — it's "choose", not "chose" 😰